packages/stream-tile/src/tile-document.ts

```diff
@@ -384,8 +385,11 @@ export class TileDocument<T = Record<string, any>> extends Stream {
     }
 
     if (content == null) {
+      const result = { header }
+      // Check if we can encode it in cbor. Should throw an error when invalid payload.
+      dagCbor.encode(result)
```

Comment:
what happens if we don't do this when we actually go and put it into ipfs?  I would have thought we'd get an error thrown anyway, from ipfs?

Reply:
In the usual scenario, when a request is sent to Ceramic node right away, yes, this change is redundant. We though expose `TileDocument.makeGenesis` as a publicly available method. It can be used for a multiquery with genesis parameter, for example, which leads to a surprising behaviour. Or, most probable case of surprising misuse is a deterministic tile + http-client. Here JSON serialization (opaque, since it is part of `fetch` API) messes with the payload and replaces `undefined` with `null`, which is not right.

Reply:
ah i see, that makes sense.  bummer.